src/safeds_stubgen/api_analyzer/_ast_visitor.py

```diff
@@ -851,6 +851,17 @@ def _get_parameter_type_and_default_value(
             elif isinstance(initializer, mp_nodes.CallExpr):
                 # Safe-DS does not support call expressions as types
                 return default_value, default_is_none
+            elif isinstance(initializer, mp_nodes.UnaryExpr):
+                value, default_is_none = self._get_parameter_type_and_default_value(initializer.expr)
+                if isinstance(value, int):
+                    return int(f"{initializer.op}{value}"), default_is_none
+                else:  # pragma: no cover
+                    msg = (
+                        f"Received an parameter {value} with an unexpected operator {initializer.op}. This parameter "
+                        f"can't be parsed."
+                    )
+                    logging.warning(msg)
```

Comment:
If we cannot handle an initializer, there should always be some warning. For example, the code could contain something like `f(p = 1 + 2)`.

You could also set the default value in the generated Safe-DS code to `unknown` and add a `TODO`. At least we know that the parameter is optional in this case.

We should never silently drop the default value, however.